src/midi.js

```diff
@@ -36,7 +47,17 @@ const midi = (notes, fileName) => {
 		}
 	});
 
-	fs.writeFileSync(fileName, file.toBytes(), 'binary');
+	let outputFile = path.join(__dirname, '../Output/', fileName + '.mid');
```

Comment:
This ends up creating the 'Output' folder in the node_modules/scribbletune folder. Thats not what one would expect or want. 

The way to test this finding is to create a dummy project which uses scribbletune. Create a new folder somewhere in your system and cd into it.

`mkdir test && cd test`

Set up a quick project by setting up a default package.json
`npm init -y`

Open the newly created package.json and point to your fork in the dependencies
```
"dependencies": {
    "scribbletune": "https://github.com/Phiph/scribbletune.git"
  },
```
Now do 
`npm i`

This will use your fork. Now create some clip and do scribble.midi with some custom folder name that you wish to see created. You ll notice the folder is not created in the root of test/ where you'd expect to see it. It ll be created in node_modules/scribbletune/ instead.

Reply:
Cheers for the review, I've changed the code to something that works. :)